numpy/core/fromnumeric.py

```diff
@@ -380,6 +380,7 @@ def choose(a, choices, out=None, mode='raise'):
     See Also
     --------
     ndarray.choose : equivalent method
+    numpy.take_along_axis : May be preferable if 'choices' is an array
```

Comment:
I think choices we usually use backticks as in ``` `choices` ```. I wonder if we should say Usually preferable, or just preferable (although there are probably cases where `choose` is faster, even if just for small arrays due to much larger overhead.